dotnet/src/support/WebDriver.Support.StrongNamed.nuspec

```diff
@@ -4,35 +4,28 @@
     <id>$packageid$</id>
     <version>$version$</version>
     <authors>Selenium Committers</authors>
-    <copyright>Copyright © 2020 Software Freedom Conservancy</copyright>
+    <copyright>Copyright © 2023 Software Freedom Conservancy</copyright>
     <owners>selenium</owners>
     <title>WebDriver Support</title>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>Provides support classes for Selenium WebDriver</summary>
     <description>
-      Selenium is a set of different software tools each with a different approach
-      to supporting browser automation. These tools are highly flexible, allowing
-      many options for locating and manipulating elements within a browser, and one
-      of its key features is the support for automating multiple browser platforms.
-      This package contains .NET support utilites and classes that users may find
-      useful in using Selenium WebDriver. These support classes are mainly intended
-      to spark ideas of what is possible with Selenium WebDriver, and may not be
-      entirely appropriate for production use.
+      Selenium is a set of different software tools each with a different approach to supporting browser automation. These tools are highly flexible, allowing many options for locating and manipulating elements within a browser, and one of its key features is the support for automating multiple browser platforms. This package contains the .NET bindings for the concise and object-based Selenium WebDriver API, which uses native OS-level events to manipulate the browser, bypassing the JavaScript sandbox, and does not require the Selenium Server to automate the browser.
     </description>
     <projectUrl>https://selenium.dev</projectUrl>
-    <repository url="https://github.com/SeleniumHQ/selenium" />
+    <repository type="GitHub" url="https://github.com/SeleniumHQ/selenium" />
     <license type="expression">Apache-2.0</license>
     <iconUrl>https://selenium.dev/images/selenium_logo_square_green.png</iconUrl>
-    <icon>images\icon.png</icon>
+    <icon>icon.png</icon>
     <tags>selenium webdriver support browser automation</tags>
+    <releaseNotes>
+      See the latest changes, included in this release, at https://raw.githubusercontent.com/SeleniumHQ/selenium/trunk/dotnet/CHANGELOG.
```

Comment:
is there a reason to prefer githubusercontent over github?
Also, can we use `$version$` here? We could use the tag instead of the branch in case we change the default branch at some point. The tag won't change and it will show the exact updates for the release:
e.g., https://github.com/SeleniumHQ/selenium/tree/selenium-4.13.0/dotnet/CHANGELOG

Reply:
> is there a reason to prefer githubusercontent over github?

To get plain text without GitHub UI. I have no preferences here.

Using `version` is very good idea. It is not clear how to deal with patches. This is list of tags:
```
selenium-4.12.0
selenium-4.12.1-dotnet
selenium-4.12.1-java
selenium-4.12.2-dotnet
selenium-4.12.3-dotnet
selenium-4.12.4-dotnet
```

I think we can mark release commit with `selenium-4.12.0-dotnet` tag to have consistent url.

Reply:
@titusfortner your opinion is needed here.

Reply:
minor versions are when we release all bindings at the same time. The patch versions are when we need to do language specific versions. We used not to tag them at all, but we were going 4-5 months between releases for a bit so it was more important to tag when we made updates to things like CDP versions in each language.

Reply:
I would prefer we use github directly so that people have context and can navigate around if the need to for any reason. Raw github is better for copying code directly, which I don't think is what we want to direct people to do.

Reply:
We decided to use static link to trunk, at least for now.